setup.cfg

```diff
@@ -24,3 +24,8 @@ install_requires =
 
 [options.packages.find]
 where = src
+
+[options.entry_points]
+console_scripts =
+    s1_cslc.py = compass.s1_cslc:main
```

Comment:
Nice! This is all it takes to do the following in terminal?
```bash
random_path$ c1_cslc some_yaml
```

Reply:
yep, and without `random_path$`.